core-api/src/entities/place.entity.ts

```diff
@@ -34,6 +35,9 @@ export class Place {
     @OneToMany(type => Meal, meal => meal.place)
     public meals: Meal[];
 
+    @OneToMany(type => Access, access => access.place)
```

Comment:
This relation has to be eager loaded. When we load a place, its access will be available as well.